packages/web-scripts/package.json

```diff
@@ -48,12 +48,11 @@
     "debug": "^4.1.1",
     "eslint": "^6.8.0",
     "jest": "^25.1.0",
-    "jest-config": "^25.1.0",
```

Comment:
jest-config was previously installed as a direct dependency to resolve a problem with ts-jest 25.3. that version seemed to introduce other larger issues, so we skipped it. I noticed that the PR was passing for 25.4, so I gave this a shot - happy to be back on a healthy path with this project.